src/MountedTraversal.js

```diff
@@ -249,8 +255,7 @@ function findAllInRenderedTreeInternal(inst, test) {
     const internal = internalInstance(inst);
     return findAllInRenderedTreeInternal(internal, test);
   }
-
-  const publicInst = inst.getPublicInstance();
+  const publicInst = inst.getPublicInstance() || inst._instance;
```

Comment:
I believe `.getPublicInstance` is a private method in React itself, which always returns `null` for SFCs, thus we fall back to `._instance`.